visivo/commands/utils.py

```diff
@@ -4,38 +4,11 @@
 from pathlib import Path
 from visivo.models.project import Project
 from visivo.models.target import Target
-from sqlalchemy import create_engine, MetaData, Table, Integer, Column, insert 
+from sqlalchemy import create_engine, MetaData, Table, Integer, Column, insert
 from visivo.parsers.core_parser import PROFILE_FILE_NAME
 from visivo.utils import load_yaml_file
 
 
-def find_default_target(project: Project, target_name: str) -> Target:
```

Comment:
We're you able to remove this because the flow of the dag is more complete now? That seems like a big win.

Reply:
Yea, I moved it over to using the dag, and setting the default on the project if it is passed in.